### PR TITLE
Add qat principles to segas

### PR DESCRIPTION
**Pull request by Dineshtj**
Is this pull request a content or a code change? (Please fill in the relevant section and delete the other)

# Code change
I can confirm:
## Accessibility considerations
- [ ] Please review the [accessibility checks for layout changes](https://github.com/UKHomeOffice/engineering-guidance-and-standards/blob/main/technical-docs/accessibility/layout-checks.md).

- [ ] This change will not change layouts, page structures or anything else that might impact accessibility

or
- [ ] This change might impact accessibility, automated aXe tests cover the impact

or
- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
(If the change might impact accessibility then please add some further information here)

# Content change 
- [x] Please review the [accessibility checks for content changes](https://github.com/UKHomeOffice/engineering-guidance-and-standards/blob/main/technical-docs/accessibility/content-checks.md).

I can confirm:
- [x ] Content does not include any code or configuration changes (excluding frontmatter information)
- [x] Content meets the content standards
e.g. [Writing a principle](https://engineering.homeoffice.gov.uk/standards/writing-a-principle/) and [Writing a standard](https://engineering.homeoffice.gov.uk/standards/writing-a-standard/)
- [x] Content is suitable to open source, i.e.:
    - Content does not relate to unreleased gov policy
    - Content does not refer to anti-fraud mechanisms
    - Content does not include sensitive business logic
- [x] Last updated date for content is correct

### CODE REVIEW

**Comment by Dineshtj** [comment]
Hey Code owners (cc @GraemeSutherlandHO ) , I have added the QAT Principles to the list of SEGAS principles. This is a content change. I have tested the page locally and ran the E2E cypress tests for accessibility. Looks good. Can you please approve?

File: docs/principles/ho-qat-principles.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+layout: principle
+order: 1
+title: Quality assurance and testing
+date: 2024-11-25
+tags:
+- quality assurance and testing
+- test engineering
```
Comment (jeff-horton-ho-sas) [inline docs/principles/ho-qat-principles.md:8]: https://engineering.homeoffice.gov.uk/tags/ is the list of currently used tags. I'd suggest adding "Testing" and "Quality" to this list. Happy for you to use the other tags, but be aware that they'll be on their own in that tag. If you do keep them these should be written as you want them displayed:
```suggestion
- Testing
- Quality
- Quality assurance and testing
- Test engineering
```

File: docs/principles/ho-qat-principles.md
Comment (jeff-horton-ho-sas) [inline docs/principles/ho-qat-principles.md:1]: I'd suggest renaming to `quality-assurance-and-testing.md` The filename (without the `.md`) is used for the url, so it should be the same as the page title.

Comment (Dineshtj) [reply docs/principles/ho-qat-principles.md:1]: Ah did not realise that. Have updated the file name. Many thanks Jeff

**Review by beseven — APPROVED** [review]
LGTM!

**Review by jeff-horton-ho-sas — APPROVED** [review]

**Pull request merged**